src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -321,6 +321,13 @@ public static FeedbackResponseCommentSearchResultBundle fromResults(
             bundle.commentGiverTable.put(comment.getId().toString(),
                     getFilteredCommentGiverName(bundle, instructorCourseIdList, response, comment, commentGiverName));
             bundle.instructorEmailNameTable.put(comment.giverEmail, commentGiverName);
+            if (!comment.lastEditorEmail.isEmpty()
+                    && !bundle.instructorEmailNameTable.containsKey(comment.lastEditorEmail)) {
```

Comment:
Use an explanatory variable to document the meaning of the full boolean expression?